### ISSUE TITLE
[Accessibility][Name, Role, Value - Notification]: Ensures every ARIA input field has an accessible name

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2123]
**User Experience:**
This issue should be resolved for good coding practice as all ARIA input field must has an accessible name.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Notification' from left navigation.
5. Run K4W tool on the page and Observe the issue. 

**Actual Result:**
ARIA input field i.e. 'All' does not has an accessible name. Screen reader read only like : All selected, selection between three items'.

This accessibility issue was found using Accessibility Insights for Web(keros), a tool that helps debug and find accessibility issues earlier. Get more information & download this tool at http://aka.ms/Keros.

**Expected Result:**
Ensures every ARIA input field has an accessible name. Screen reader should read like : Notification_All button selected, selection between three items'.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
[MAS4.1.2_Notification_All Button.zip](https://github.com/microsoft/BotFramework-Composer/files/4266321/MAS4.1.2_Notification_All.Button.zip)

**Comment by corinagum** [comment]
![MAS4 1 2_Notification_All Button](https://user-images.githubusercontent.com/14900841/75577176-4d3f5d00-5a16-11ea-9fe5-87c49d1070c6.png)
![MAS4 1 2_Notification_All Button_K4W](https://user-images.githubusercontent.com/14900841/75577177-4dd7f380-5a16-11ea-990a-e17920127223.png)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
